src/elfpy/strategies/simple_short.py

```diff
@@ -0,0 +1,42 @@
+# pylint: disable=duplicate-code
```

Comment:
Can you rename this file so that it is meaningfully different from `single_short.py`?